src/script.js

```diff
@@ -222,6 +222,21 @@ export function getLocale() : LocaleType {
         if (COUNTRY_LANGS.hasOwnProperty(country) && COUNTRY_LANGS[country].indexOf(lang) !== -1) {
             // $FlowFixMe
             return { country, lang };
+        } else {
+            // We infer country from language if there is only one possible country match
+            const possibleCountries = Object.keys(COUNTRY_LANGS).filter(x => COUNTRY_LANGS[x].some(y => y === lang));
+            
+            if (possibleCountries.length !== 1) {
+                break;
+            }
+
+            const possibleCountry = possibleCountries[0];
+            const languages = COUNTRY_LANGS[possibleCountry];
```

Comment:
@amyegan Do we have to determine language here since we are already filtering by `lang` to get possible country?  Wouldn't we just use `lang`?

Reply:
Yeah this is a fair point